arc.tex

```diff
@@ -0,0 +1,110 @@
+%ENclave text from KT  https://docs.google.com/document/d/1apNSWtIpbS7aCitaF_K0JXLAAY9PvCEmmvnNvHmecdk/edit#heading=h.nwgy0freb3zv
+
+
+\section{System architecture} \label{sec:arc}
+The overall system architecture is available in \citeds{LDM-148}.
+Details on the \gls{USDF} specifications are given in \citeds{DMTN-189}.
+\action{GS,FH}{March 2022}{Should we have Specs like \gls{DMTN}-189 for UK and FRdF ?}
+
+\secref{sec:desc} gives a high level overview of the system, architecturally we look
+at this as a set of enclaves.
+As images are processed in the Prompt and Offline Production enclaves, their resulting data products are stored in the \gls{Archive} enclave and made available to the DAC enclave where data rights holders can access and analyze them.
+In addition, Rubin Observatory staff will use the Development/Integration enclave to maintain the Observatory's software tools and systems and to develop new versions of them.
+
+These enclaves are further described here and for each a series of subsections explore :
+
+
+\begin{enumerate}
+\item Threats and Security infrastructure
+\item Disaster recovery
+\end{enumerate}
+
+\subsection{Prompt \gls{Enclave}}
+
+The Prompt enclave receives images from the Observatory facilities in Chile via a Long Haul Network connection.
```

Comment:
Should we explicitly refer to Chile as "the summit enclave"?

Reply:
Data will also flow (physically, if not logically) through equipment at the Base.